lib/tasks/deployment/20230316205540_fix_changing_of_source_file_status_enum_value.rake

```diff
@@ -1,15 +0,0 @@
-namespace :after_party do
```

Comment:
@littleforest is there any side effect if we delete this after party tasks? wondering if it would be best to keep them, but make them NOOP

Reply:
No, there's no issue. They've already run on prod, and would just get run in dev by a new developer, so doesn't matter if they are missing or noop. We have the task id in the `task_versions` table, so we could always use it to find the file in git if needed (seems unlikely though).